src/app/views/sidebar/resource-explorer/collection/Paths.tsx

```diff
@@ -1,105 +1,112 @@
 import { ResourcePath } from '../../../../../types/resources';
 import { formatScopeLabel, scopeOptions } from './collection.util';
 import { PERMS_SCOPE } from '../../../../services/graph-constants';
+import pathStyles from './Paths.styles';
 
 interface IPathProps {
   resources: ResourcePath[];
-  columns: IColumn[];
+  columns: { key: string, name: string, fieldName: string, minWidth: number, maxWidth: number, isResizable: boolean }[];
   isSelectable?: boolean;
   onSelectionChange?: (selectedItems: ResourcePath[]) => void;
 }
 
-const scopeLabelClass = mergeStyles({
-  backgroundColor: '#616161',
-  color: 'white',
-  padding: '4px 12px',
-  borderRadius: '16px',
-  fontSize: '12px',
-  display: 'inline-block',
-  textAlign: 'center'
-});
-
-export default class Paths extends Component<IPathProps> {
-  readonly _selection: Selection | null = null;
-
-  constructor(props: IPathProps) {
-    super(props);
+const Paths: React.FC<IPathProps> = ({ resources, columns, isSelectable, onSelectionChange }) => {
+  const styles = pathStyles();
+  const [selection, setSelection] = React.useState<Set<ResourcePath>>(new Set());
+  const [allSelected, setAllSelected] = React.useState(false);
 
-    if(props.isSelectable) {
-      this._selection = new Selection({
-        onSelectionChanged: () => {
-          const selected = this._selection!.getSelection() as ResourcePath[];
-          props.onSelectionChange && props.onSelectionChange(selected);
-        }
-      });
+  const handleSelectionChange = (item: ResourcePath) => {
+    const newSelection = new Set(selection);
+    if (newSelection.has(item)) {
+      newSelection.delete(item);
+    } else {
+      newSelection.add(item);
     }
-  }
-
-  readonly renderCustomCheckbox = (props: any): JSX.Element => {
-    return (
-      <div style={{ pointerEvents: 'none' }}>
-        <Checkbox checked={props ? props.checked : false} />
-      </div>
-    );
+    setSelection(newSelection);
+    onSelectionChange && onSelectionChange(Array.from(newSelection));
+    setAllSelected(newSelection.size === resources.length);
   };
 
-  readonly renderItemColumn = (item: ResourcePath, index: number | undefined, column: IColumn | undefined) => {
+  const handleSelectAllChange = () => {
+    if (allSelected) {
+      setSelection(new Set());
+      onSelectionChange && onSelectionChange([]);
+    } else {
+      const newSelection = new Set(resources);
+      setSelection(newSelection);
+      onSelectionChange && onSelectionChange(Array.from(newSelection));
+    }
+    setAllSelected(!allSelected);
+  };
 
-    if (column) {
-      if (column.key === 'scope') {
-        return <Label className={scopeLabelClass}>
+  const renderItemColumn = (
+    item: ResourcePath,
+    column: { key: string, name: string, fieldName: string, minWidth: number, maxWidth: number, isResizable: boolean }
+  ) => {
+    if (column.key === 'scope') {
```

Comment:
Is there a way to widen this column? it looks a little squeezed..
![image](https://github.com/user-attachments/assets/e05b8072-a818-476b-a3c0-9c959f8eb1e8)

Reply:
Is this a responsive issue? Is it on the My API collections page?

Reply:
No, it appears that way on all screen sizes. 
Yes it's on the My API collection page and the Edit Collections and Edit scope page

Reply:
Hmmm. for some reason I can't replicate this on both large or small screen. Reaching out to screen share

Reply:
This is how it looks like for me
![screenshot-1738147601026](https://github.com/user-attachments/assets/6ee55040-f203-4d5d-b7ad-f599a5734828)

Reply:
Me too
![image](https://github.com/user-attachments/assets/96b44065-81f8-4b0e-80f1-c5e19ea648b9)

Reply:
Removing this value on that `td` element fixes for me
![image](https://github.com/user-attachments/assets/c8823338-fb7a-4c51-bc53-e8460fea699c)

![image](https://github.com/user-attachments/assets/3c38ff55-1ce8-4151-93f2-12d351e4190c)